lib/stat_tracker.rb

```diff
@@ -0,0 +1,19 @@
+require 'CSV'
+
+class StatTracker
+  def self.from_csv(locations)
+    CSV.foreach(locations[:games], headers: true, header_converters: :symbol) do |row|
+      away_team_id = row[:away_team_id].to_i
+    end
+  end
+
+  def self.count_of_teams(locations)
```

Comment:
This still spikes my interest as far as being a class method (I feel like it should be an instance method)... I would like you to show me how this works as far as scalability/ big picture.